src/ctap/data_formats.rs

```diff
@@ -281,23 +281,23 @@ impl Extensions {
     pub fn has_make_credential_hmac_secret(&self) -> Result<bool, Ctap2StatusCode> {
         self.0
             .get("hmac-secret")
-            .map(read_bool)
+            .map(|b| extract_bool(b.clone()))
```

Comment:
I've added a comment on `extract_bool`. If it works, you should be able to simply write `extract_bool` removing the need for `clone`.

Reply:
After discussion, the change is not worth it. See the rationale in the `extract_bool` comment.